sources/osg/FrameBufferObject.js

```diff
@@ -216,7 +218,14 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
         var gl = this._gl;
 
         // apply on unit 1 to init it
-        state.applyTextureAttribute( 1, texture );
+        // make sure we do bind it whatever state stack
+        // texture is cached
+        state.applyTextureAttribute( 1, texture, true );
```

Comment:
like above, Framebuffer like VAO, if texture not really binded (not cache) just before, it's not framebuffer associated.
That's the simple way to make sure we do bind it.

It's part of the enforce better log/sentry by forcefully following spec, not a case I reproduced.
Doesn't cost much to add, but allow to make sure we're doing the right thing